app-shells/fish/fish-2.6.0.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
```

Comment:
Please remove this line. See commit 61b861acd7b49083dab687e133f30f3331cb7480 for the reason.